resources/views/components/radio-buttons.blade.php

```diff
@@ -1,7 +1,7 @@
 @foreach($options as $value => $atts)
 <div class="field">
     @php($hint = isset($atts['hint']) ? $name . '-' . $value . '-hint' : '')
-    <input {!! $attributes !!} type="radio" name="{{ $name }}" id="{{ $name }}-{{ $value }}" value="{{ $value }}" {!! $describedBy($hint) ? 'aria-describedby="' . $describedBy($hint) . '"' : '' !!} @checked($checked === $value) {!! $invalid ? 'aria-invalid="true"' : '' !!} />
+    <input {!! $attributes !!} type="radio" name="{{ $name }}" id="{{ $name }}-{{ $value }}" value="{{ $value }}" {!! $describedBy($hint) ? 'aria-describedby="' . $describedBy($hint) . '"' : '' !!} @checked($checked == $value) {!! $invalid ? 'aria-invalid="true"' : '' !!} />
```

Comment:
Could a test be added for this. Assuming this came from an actual issue, the code could probably be pulled from there to use as a test case.